leaky_relu added to jax frontend

VikramxD opened the pull request.

sherry30 commented on ivy/functional/frontends/jax/nn/non_linear_activations.py:
```diff
@@ -6,3 +6,17 @@ def relu(x):
+leaky_relu.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
+
+
+def gelu(x):
```
remove this

VikramxD replied:
removed

sherry30 reviewed (changes requested):
Hi,

I think you've accidentally added `gelu`, can you kindly remove that.
other than that if you see the documentation of `leaky_relu` [here](https://jax.readthedocs.io/en/latest/_autosummary/jax.nn.leaky_relu.html#jax.nn.leaky_relu), the argument is called `negative_slope` and not `alpha`. so you will have to change that argument name.

Thanks

VikramxD commented:
pr for #2823 from the todolist issue #1515  

1. Implemented leaky_relu in  ivy/functional/frontends/jax/nn/non_linear_activations.py
2. added tests appropriate to leaky_relu functions in ivy_tests/test_ivy/test_frontends/test_jax/test_activations.py

VikramxD commented:
@sherry30 this pr is ready for re review please do tell if anything else needs to be changed

sherry30 reviewed (changes requested):
Hi,

Can you also add `negative_slope` argument in your test.
Ideally the test function should test all arguments of the function.

Thanks.

VikramxD commented:
@sherry30 changes done accordingly
